www/.gitignore

```diff
@@ -34,3 +34,14 @@ src/cache
 
 # Env variables
 .env*
+
+# Dependency directories
+node_modules/
+jspm_packages/
+
+# Optional npm cache directory
+.npm
+
+# System Files
+.DS_Store
+Thumbs.db
```

Comment:
the `.DS_Store` is already in top level .gitignore - https://github.com/gatsbyjs/gatsby/blob/f671f8632c8253eca4208b9aafd13bf89799e990/.gitignore#L42

so I think we could remove this from here, and add `Thumbs.db` to top level ignore file as well

the `npm`/`jspm` ones are fine to keep here, because we can't use `npm` in things that use `yarn`'s workspaces (so mostly `packages` directory and root of monorepo)